azuredevops/internal/service/build/resource_build_definition.go

```diff
@@ -180,6 +180,11 @@ func ResourceBuildDefinition() *schema.Resource {
 							Optional: true,
 							Default:  "",
 						},
+						"report_build_status": {
+							Type:     schema.TypeBool,
```

Comment:
Is this only for bitbucket or do all sources have this?

Reply:
All resource. This is pipeline configuration